### ISSUE TITLE
NetworkId parameter in list-ip-adresses

### ISSUE THREAD

**Issue by inge4pres** [CenturyLinkCloud/clc-go-cli#85]
Hi,

after successful login
```
$ ~/bin/clc.exe login --user user.name --password sup3rs3cretPWD
Logged in as user.name.
```
I try to get the IPs in a network and get such error
```
$ ~/bin/clc.exe network list-ip-addresses --data-center VA1 --network-id xxxxxxxzzzzzzzzyyyyy1234
Unknown option or argument: `NetworkId`.
```

**Comment by inge4pres** [comment]
Same goes with NetworkName
```
$ ~/bin/clc.exe network list-ip-addresses --data-center VA1 --network-name "NET_NAME"
Unknown option or argument: `NetworkName`.
```

**Comment by chrislittle** [comment]
This looks like a bug

@richardcase I can't get this to work either.  can you check the code?

if I use --network (instead of network-name or network-id) it actually goes forward and wants a --type (which is supposed to be optional).  even using that then I get a 404.  
```
Error occured while sending request to API. Status code: 404. Server says: The requested resource does not exist.
```

@inge4pres you have the logic right in my view.

**Comment by richardcase** [comment]
Sure, I'll check the code and let you both know.

**Comment by richardcase** [comment]
@inge4pres & @chrislittle : i've had a look at the code and i can see there is a bug. I am in the process of migrating to a new laptop but i will get it fixed asap and will let you both know.

**Comment by inge4pres** [comment]
I'm on v1.1.0-rc.1

**Comment by chrislittle** [comment]
thanks @inge4pres as @richardcase noted he's going to dig into the code and fix this up.  if you find anything else let us know as we appreciate the feedback

**Comment by richardcase** [comment]
Fixed in release v1.1.0-rc.2. https://github.com/CenturyLinkCloud/clc-go-cli/releases/tag/v1.1.0-rc.2
